addons/addon-base-raas-appstream/packages/base-raas-appstream-services/lib/plugins/__tests__/aws-account-mgmt-plugin.test.js

```diff
@@ -0,0 +1,165 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const ServicesContainer = require('@aws-ee/base-services-container/lib/services-container');
+
+// Mocked dependencies
+jest.mock('@aws-ee/base-services/lib/plugin-registry/plugin-registry-service');
+const PluginRegistryService = require('@aws-ee/base-services/lib/plugin-registry/plugin-registry-service');
+
+jest.mock('@aws-ee/base-services/lib/settings/env-settings-service');
+const SettingsServiceMock = require('@aws-ee/base-services/lib/settings/env-settings-service');
+
+jest.mock('@aws-ee/base-raas-services/lib/environment/service-catalog/environment-sc-service');
+const EnvironmentScServiceMock = require('@aws-ee/base-raas-services/lib/environment/service-catalog/environment-sc-service');
+
+jest.mock('@aws-ee/base-raas-services/lib/indexes/indexes-service');
+const IndexesServiceMock = require('@aws-ee/base-raas-services/lib/indexes/indexes-service');
+
+const plugin = require('../aws-account-mgmt-plugin');
+
+// Tested Functions: getActiveNonAppStreamEnvs
+describe('awsAccountMgmtPlugin', () => {
+  let container;
+  let settings;
+  let environmentScService;
+  let indexesService;
+  beforeEach(async () => {
+    // Initialize services container and register dependencies
+    container = new ServicesContainer();
+    container.register('pluginRegistryService', new PluginRegistryService());
+    container.register('settings', new SettingsServiceMock());
+    container.register('environmentScService', new EnvironmentScServiceMock());
+    container.register('indexesService', new IndexesServiceMock());
+
+    await container.initServices();
+    settings = await container.find('settings');
+    environmentScService = await container.find('environmentScService');
+    indexesService = await container.find('indexesService');
+  });
+
+  describe('getActiveNonAppStreamEnvs', () => {
+    const requestContext = { principalIdentifier: { uid: 'u-testuser' } };
+    it('should return empty list if AppStream is disabled', async () => {
+      // BUILD
+      const awsAccountId = 'sampleAwsAccountId';
+      settings.getBoolean = jest.fn(() => {
+        return false;
+      });
+      const expected = [];
+
+      // OPERATE
+      const retVal = await plugin.getActiveNonAppStreamEnvs({ awsAccountId }, { requestContext, container });
+
+      // TEST
```

Comment:
Lol, sorry for this nitpick: But we've been going with `CHECK` instead of `TEST` for this comment